DZ#9terrform-2

igorborodko opened the pull request:
Задания выполнил, кроме ДЗ с звездочкой.

# Выполнено ДЗ №

 - [x] Основное ДЗ
 - [ ] Задание со *

## В процессе сделано:
 - Пункт 1
 - Пункт 2

## Как запустить проект:
 - Например, запустить команду X в директории Y

## Как проверить работоспособность:
 - Например, перейти по ссылке http://localhost:8080

## PR checklist
 - [ ] Выставил label с номером домашнего задания
 - [ ] Выставил label с темой домашнего задания

igorborodko committed:
DZ#9terrform-2

Задания выполнил, кроме ДЗ с звездочкой.

mrgreyves reviewed (changes requested):
Здравствуйте!)
У вас отсутствуют тесты
Имею ввиду "ямл файлы" в директории .github/workflows
Может быть вы забыли смержить их к себе в ветку перед выполнением дз?
Это нужно поправить
ДЗ возвращаю на доработку

igorborodko commented:
Здравствуйте!
тесты присутствуют в директории \igorborodko_infra\.github\workflows
run-tests-2022-08.yml и auto-assign.yml
Проверьте пожалуйста, все есть.

igorborodko commented:
Тест ищет его по другому пути:
configuration-path: .github/auto_assign.yml

igorborodko committed:
Переложил auto-assign

Переложил auto-assign на уровень выше в gorborodko_infra\.github

Тест ищет его по другому пути:
configuration-path: .github/auto_assign.yml
